snap/implicit.go

```diff
@@ -29,13 +29,13 @@ var implicitSlots = []string{
 	"network-bind",
 	"network-control",
 	"network-observe",
-	"snap-control",
+	"snapd-control",
 	"system-observe",
 	"timeserver-control",
 	"timezone-control",
 	// XXX: those two should perhaps not be added by default
 	"unity7",
-	"x",
+	"x11",
```

Comment:
Can we run through that list in tests and double-check the names actually exist?